package.json

```diff
@@ -105,6 +105,7 @@
         "qunit": "2.19.1",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
+        "react-markdown": "^8.0.5",
```

Comment:
Move to dependencies and not dev devDependencies and run `npm i` again

Reply:
Done.